### PR TITLE
Remove TestPerson #310

### LINKED ISSUE #310: Remove TestPerson
Unless we specifically write unit tests for it to ensure it satisfied all of the contracts of a `ReadOnlyPerson`, having it around is just another potential source of bugs (e.g. violation of LSP) and just a general code smell.

(e.g., for example, I think `name = new Name(); name.getName()` returns null, which a ReadOnlyPerson should never do.)

### PR DESCRIPTION
**Pull request by chao1995**
Fixes #310 

Based on [previous PR v6](https://github.com/se-edu/addressbook-level4/pull/335). Ready for review.

### CODE REVIEW

**Comment by CanIHasReview** [comment]
# v1

@chao1995 submitted v1 for review.

* [v1 1/2] [Remove TestPerson](https://github.com/se-edu/addressbook-level4/pull/373/commits/17a4e4b3938c4c7f5da7cf46cdebd9b17f986353)
* [v1 2/2] [PersonUtil: remove magic strings in getAddCommand\(Person\)](https://github.com/se-edu/addressbook-level4/pull/373/commits/1e02b657310b86b3084c208bac18bdee9883e86f)

([:books: Archive](https://github.com/se-edu/addressbook-level4/compare/a61db1bf27f62cc549eb79be0a4598f2f1994b9b...1e02b657310b86b3084c208bac18bdee9883e86f))
<details>
<summary>Checkout this PR version locally</summary>

```shell
git fetch https://github.com/se-edu/addressbook-level4.git refs/pr/373/1/head:BRANCHNAME
```
where `BRANCHNAME` is the name of the local branch you want to fetch this PR to.

</details>

File: src/test/java/seedu/address/testutil/PersonBuilder.java [hunk 1]
```diff
@@ -1,28 +1,43 @@
...
+import seedu.address.model.util.SampleDataUtil;
 
 /**
- *
+ * A utility class to help with building Person objects.
  */
 public class PersonBuilder {
 
-    private TestPerson person;
+    public static final String DEFUALT_NAME = "Alice Pauline";
```
Comment (eugenepeh) [inline src/test/java/seedu/address/testutil/PersonBuilder.java]: > DEFUALT

Typos?

**Comment by CanIHasReview** [comment]
# v2

@chao1995 submitted v2 for review.

* [v2 1/2] [Remove TestPerson](https://github.com/se-edu/addressbook-level4/pull/373/commits/58b7fb3df92aeaa25b695d345059a2793b1dfb6a)
* [v2 2/2] [PersonUtil: remove magic strings in getAddCommand\(Person\)](https://github.com/se-edu/addressbook-level4/pull/373/commits/27439b724b7df1cde2b39a879106ca13e4e40a77)

([:books: Archive](https://github.com/se-edu/addressbook-level4/compare/07367fab5054768f3243fb55eba1367d3d67a39b...27439b724b7df1cde2b39a879106ca13e4e40a77)) ([:chart_with_upwards_trend: Interdiff between v1 and v2](https://github.com/se-edu/addressbook-level4/compare/403b44f6f72870ad3bbddbc50cb60b84cb1f5803~1...403b44f6f72870ad3bbddbc50cb60b84cb1f5803))
<details>
<summary>Checkout this PR version locally</summary>

```shell
git fetch https://github.com/se-edu/addressbook-level4.git refs/pr/373/2/head:BRANCHNAME
```
where `BRANCHNAME` is the name of the local branch you want to fetch this PR to.

</details>

**Comment by CanIHasReview** [comment]
# v3

@chao1995 submitted v3 for review.

* [v3 1/1] [Remove TestPerson](https://github.com/se-edu/addressbook-level4/pull/373/commits/58b7fb3df92aeaa25b695d345059a2793b1dfb6a)

([:books: Archive](https://github.com/se-edu/addressbook-level4/compare/07367fab5054768f3243fb55eba1367d3d67a39b...58b7fb3df92aeaa25b695d345059a2793b1dfb6a)) ([:chart_with_upwards_trend: Interdiff between v2 and v3](https://github.com/se-edu/addressbook-level4/compare/d54eb13e3eaa42c4c0f27aa855e53a01b2e317de~1...d54eb13e3eaa42c4c0f27aa855e53a01b2e317de))
<details>
<summary>Checkout this PR version locally</summary>

```shell
git fetch https://github.com/se-edu/addressbook-level4.git refs/pr/373/3/head:BRANCHNAME
```
where `BRANCHNAME` is the name of the local branch you want to fetch this PR to.

</details>

**Comment by chao1995** [comment]
@eugenepeh Dropped the second commit and created another PR for it. Please help review again. Thanks!

File: src/test/java/guitests/AddCommandTest.java [hunk 2]
```diff
@@ -7,16 +7,17 @@
 import guitests.guihandles.PersonCardHandle;
```
Comment (damithc) [inline src/test/java/guitests/AddCommandTest.java]: Commit message:

>Let's remove TestPerson and use Person instead. With default fields,
it's still easy to construct a person using PersonBuilder. Let's also
add PersonUtil to hold utility methods for Person, like getAddCommand.

`construct a Person`?
`getAddCommand(Person)`?

File: src/test/java/seedu/address/testutil/PersonUtil.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
+package seedu.address.testutil;
+
+import seedu.address.model.person.Person;
+
+public class PersonUtil {
```
Comment (yamgent) [inline src/test/java/seedu/address/testutil/PersonUtil.java]: Could use a header comment here for the util class.

**Comment by CanIHasReview** [comment]
# v4

@chao1995 submitted v4 for review.

* [v4 1/1] [Remove TestPerson](https://github.com/se-edu/addressbook-level4/pull/373/commits/1b07d1b6363567346760ae38bbfd392ce5ff8785)

([:books: Archive](https://github.com/se-edu/addressbook-level4/compare/aa9b3b392f5264260d60403c0c9715171692b1d4...1b07d1b6363567346760ae38bbfd392ce5ff8785)) ([:chart_with_upwards_trend: Interdiff between v3 and v4](https://github.com/se-edu/addressbook-level4/compare/7cc61b0059b9ea4ad560ad066047145bcff9159e~1...7cc61b0059b9ea4ad560ad066047145bcff9159e))
<details>
<summary>Checkout this PR version locally</summary>

```shell
git fetch https://github.com/se-edu/addressbook-level4.git refs/pr/373/4/head:BRANCHNAME
```
where `BRANCHNAME` is the name of the local branch you want to fetch this PR to.

</details>

**Pull request merged**